velox/docs/configs.rst

```diff
@@ -129,6 +129,14 @@ Generic Configuration
      - 0
      - If it is not zero, specifies the time limit that a driver can continuously
        run on a thread before yield. If it is zero, then it no limit.
+   * - prefixsort_max_normalized_key_size
+     - integer
+     - 2147483647
+     - The max bytes number of normalized keys for each row in the prefix-sort. The default value is max int, and set to 0 to disable prefix-sort.
+   * - prefixsort_threshold
+     - integer
+     - 130
+     - The minimum number of rows that can trigger the prefix-sort. The default value depends on benchmark.
```

Comment:
Minimum number of rows to use prefix-sort. The default value has been derived using micro-benchmarking.